special-pages/pages/new-tab/app/components/Layout.js

```diff
@@ -0,0 +1,9 @@
+import { h } from 'preact'
+
+export function Centered ({ children }) {
+    return (
+        <div class="layout-centered">
```

Comment:
Is the plan to eventually adopt <Centered> across the board and move `.layout-centered` out of the global namespace?

Reply:
I wanted to do that, but then I realised that I'm using it elsewhere as an additional class... not sure what's best right now, let's revisit when there's some pain :)